File: problem_thirty_seven.py
inp1 = int(input("enter the length:"))
inp2 = input("Enter the singal number:")
def MyFunction(inp,inp2):
    ind = 1
    while inp >= ind: 
        print(inp2*ind)
        ind+=1
        if inp == ind :
            while inp > 0:
                print(inp*inp2)
                inp -=1
                



MyFunction(inp1,inp2)
    
   


   

File: problem_thirty_seven.js
var inp = 5
var inp1  = "*"
function MyFunction(inp,inp1){
    var val = 1
    while (inp >= val){
        console.log(inp1*val)
        val +=1
    }if (inp == val){
        while (inp > 0){
            console.log(inp1* inp)
            inp -=1
        }
    }
}
MyFunction(inp,inp1)